.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x]
```

Comment:
if we go down to just one version, should we just set at top level?

Alternatively could add `16` and `18` and verify it builds.

Reply:
Added a few versions